aimmo-game/simulation/levels/completion_checks.py

```diff
@@ -0,0 +1,16 @@
+def check_complete(self, game_state):
+    try:
+        main_avatar = game_state.get_main_avatar()
+    except KeyError:
+        return False
+
+    return main_avatar.score > 1000
+
+COMPLETION_CHECKS = {
```

Comment:
can't that be templated somehow and linked to the max number of levels defined somewhere?

Reply:
The idea is that if I remove the completion check, it will just use the default one, that is `return False`.